solr/core/src/test/org/apache/solr/cloud/HttpPartitionTest.java

```diff
@@ -563,25 +541,29 @@ protected void assertDocsExistInAllReplicas(
     }
   }
 
-  protected SolrClient getHttpSolrClient(Replica replica, String coll) {
+  protected SolrClient getHttpSolrClient(Replica replica, String collection) {
```

Comment:
if it was `coll` everywhere in the code base, then I'd be a fan of `coll`.  Since I see `collection`, i made that change....    One thing I thought about was going through and at least in the tests, we use `collection`, `testCollectionName` and `collectionName` and getting that to one value..